lib/meataxe.gi

```diff
@@ -3096,11 +3096,11 @@ local cf,dim,b,den,sub,i,s,q,found,qb;
             Add(sub,TriangulizedMat(b));
             den:=dim;
           else
-            TriangulizeMat(s);
+            s:=TriangulizedMat(s);
```

Comment:
Isn't the old version better?
With `qb` below you changed the code the other way round.